infra/core/ai/bingSearch/bingSearch.tf

```diff
@@ -23,4 +23,10 @@ resource "azurerm_resource_group_template_deployment" "bing_search" {
   # this ensures the keys are up-to-date
   name            = "bingsearch-${filemd5(local.arm_file_path)}"
   deployment_mode = "Incremental"
+}
```

Comment:
This may be an issue when ENABLE_WEB_SEARCH=false is set.. as this key index in KV will never get created since this module gets skipped.. Please be sure to test to make sure the webapp AppService doesn't generate errors on startup if this key does not exist in the KeyVault.

Reply:
I did get a BINGSEARCH-KEY was not found in this key vault error. Looking for a way to create the key even if it won't be used

Reply:
Bing Search Key is now being set to "" if ENABLE_WEB_CHAT is set to false